packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.types.ts

```diff
@@ -98,18 +322,37 @@ export interface IContextualMenuItemStyleProps {
   theme: ITheme;
 
   /**
-   * Accept custom classNames
+   * Additional CSS class(es) to apply to the ContextualMenuItem.
    */
   className?: string;
 
   // Insert ContextualMenuItem style props below
+  disabled: boolean;
+  expanded: boolean;
+  checked: boolean;
+  isAnchorLink: boolean;
+  knownIcon: boolean;
+  itemClassName?: string;
+  dividerClassName?: string;
+  iconClassName?: string;
+  subMenuClassName?: string;
+  primaryDisabled?: boolean;
```

Comment:
I can perhaps use `Pick<T>` here?

Reply:
I don't really think I can upon revisit. There doesn't seem to be anything in common as the interfaces are currently written.